### PR TITLE
Improve Auth Section

### PR DESCRIPTION
**Pull request by estevaofay**
## Problem
Problems listed here
https://www.notion.so/getmentum/Add-corrections-to-Authentication-docs-58613cdf65da4ba8889889e9a53b18bc

## Solution

Fixing openapi to include improvements

## Testing Done

- [x] Spec builds using `redoc-cli serve`
- [x] Previewed changes locally

### CODE REVIEW

File: openapi/openapi.yml [hunk 1]
```diff
@@ -189,10 +191,21 @@ paths:
...
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    message:
+                      description: The response message
+                      type: string
+                      example: Unauthorized
   /api/accessToken/logout:
       post:
```
Comment (dosvath) [inline openapi/openapi.yml:206]: do we need a `security` section for this `/logout` endpoint?

**Review by dosvath — COMMENTED** [review]
Thanks for the updates!

File: openapi/openapi.yml (hunk 1 above)
Comment (estevaofay) [reply openapi/openapi.yml:206]: Yes, the top level security section applies to every endpoint.

This specific section is to clear it for this endpoint

Comment (dosvath) [reply openapi/openapi.yml:206]: I meant to say `/logout` has no `security` section specified under `post`, but `refresh` and the other endpoint do, can you confirm why this is intended?

Comment (dosvath) [reply openapi/openapi.yml:206]: Have you tested we can use `/logout` without authentication?

**Review by dosvath — APPROVED** [review]
LGTM

**Pull request merged**